### PR TITLE
Add CI

### PR DESCRIPTION
**Pull request by pezholio**
This adds a Github action to run our tests in CI. Eventually, we'll probably a Docker image to run the tests in, but this is a start.

### CODE REVIEW

File: script/ci [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+set -e
+
+export DATABASE_URL="postgres://$POSTGRES_USER:$POSTGRES_PASSWORD@$POSTGRES_HOST:$POSTGRES_PORT/$POSTGRES_DB"
+
+cd "$(dirname "$0")/.."
+
+echo "==> Installing dependencies..."
+
+npm install
```
Comment (Gweaton) [inline script/ci:14]: Do we want to use `npm ci` here?

> This command is similar to npm install, except it's meant to be used in automated environments such as test platforms, continuous integration, and deployment -- or any situation where you want to make sure you're doing a clean install of your dependencies.

Taken from: https://docs.npmjs.com/cli/v8/commands/npm-ci

File: script/ci [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+echo "==> Running database migrations"
+
+npm run typeorm migration:run
+
+echo "==> Building assets..."
+
+npm run build:assets
```
Comment (Gweaton) [inline script/ci:22]: Do we need to build the NestJS app here too with `npm run build`?

**Review by Gweaton — COMMENTED** [review]
Nice work, just a couple of comments

File: script/ci (hunk 2 above)
Comment (pezholio) [reply script/ci:22]: `npm run start:test` will build the app anyway. We'll probably want to do a production build for deployment though, but that can come later and in a different pipeline.

**Review by Gweaton — APPROVED** [review]

File: script/ci (hunk 1 above)
Comment (Gweaton) [reply script/ci]: Yeah, I only found out about it just now looking at the [starter workflows](https://github.com/actions/starter-workflows/blob/main/ci/node.js.yml)!

**Pull request merged**